### DISCUSSION TITLE
v11.2.1 added battery indicators for Ring security system products, can these be toggled off?

### DISCUSSION THREAD

**Discussion by eoddc5** [dgreif/ring#1031] [Ideas]
I just updated to v11.2.1, and the battery indicator "lights" for ring security system (base station and keypad) just showed up - would rather make this an option in settings that can be hidden, as they are not required or necessary for me and they now always appear on the home main app page because theyre "lights"

any way to add that in?

**Comment by dgreif** [comment]
Just like every device in the plugin, you can hide these using the `hideDeviceIds` option. The id for each device is logged when homebridge starts. My default is to show _everything_ and let users hide the things they don't wan't/care about.

**Reply by mzuniga51** [reply]
Hi.  For me it would be better to have an option to hide them all.  I have about 50+ devices so exclude them individually is time consuming.  The problem for me arises when exposing the devices to Alexa where the device itself and its battery status appear as two individual devices sharing the same name which makes Alexa unable to control them by voice.
